package.json

```diff
@@ -56,28 +56,24 @@
   "directories": {
     "doc": "src/docs/md"
   },
-  "peerDependencies": {
-    "@angular/cdk": "^14.0.0",
-    "@angular/core": "^14.0.0",
-    "@angular/material": "^14.0.0"
-  },
+  "peerDependencies": {},
   "devDependencies": {
-    "@angular-devkit/build-angular": "^14.2.9",
-    "@angular/animations": "^14.2.10",
-    "@angular/cdk": "^14.2.7",
-    "@angular/cli": "^14.2.9",
-    "@angular/common": "^14.2.10",
-    "@angular/compiler": "^14.2.10",
-    "@angular/compiler-cli": "^14.2.10",
-    "@angular/core": "^14.2.10",
-    "@angular/flex-layout": "^14.0.0-beta.41",
-    "@angular/forms": "^14.2.10",
-    "@angular/material": "^14.2.7",
-    "@angular/platform-browser": "^14.2.10",
-    "@angular/platform-browser-dynamic": "^14.2.10",
-    "@angular/router": "^14.2.10",
-    "@brightlayer-ui/angular-components": "^7.0.3",
-    "@brightlayer-ui/angular-themes": "^8.0.0",
+    "@angular-devkit/build-angular": "^15.2.8",
+    "@angular/animations": "^15.2.9",
+    "@angular/cdk": "^15.2.9",
+    "@angular/cli": "^15.2.8",
+    "@angular/common": "^15.2.9",
+    "@angular/compiler": "^15.2.9",
+    "@angular/compiler-cli": "^15.2.9",
+    "@angular/core": "^15.2.9",
+    "@angular/flex-layout": "^15.0.0-beta.42",
+    "@angular/forms": "^15.2.9",
+    "@angular/material": "^15.2.9",
+    "@angular/platform-browser": "^15.2.9",
+    "@angular/platform-browser-dynamic": "^15.2.9",
+    "@angular/router": "^15.2.9",
+    "@brightlayer-ui/angular-components": "9.0.0-alpha.1",
+    "@brightlayer-ui/angular-themes": "9.0.0",
```

Comment:
Please use Caret Version Range (^) in the angular-themes version. It should be like "@brightlayer-ui/angular-themes": "^9.0.0"